### ISSUE TITLE
Accessibility: auto scroll causing issues for screen magnification

### ISSUE THREAD

**Issue by connorscarolyns** [cds-snc/platform-forms-client#4688]
Screen magnification users often have narrow fields of focus. Screens that auto-scroll can take their focus away and make it difficult to fill out forms.

Assistive technology
OS Magnification

Browser
Edge

https://app.makeitfable.com/share/670e9d94fa0a0c5fe7e051f8

**Comment by thiessenp-cds** [comment]
An interesting UI challenge. A user with low vision can struggle with auto-focus while other users benefit from it. A similar ticket is enabling speech-to-text for users with a mobility impairment, while other users might find the added interface for this confusing (see #4689).  The only solution I can think of is to add a new accessibility settings interface (example below). 
What do you think @connorscarolyns? And if so, perhaps in a future itteration?
![Screenshot 2024-12-02 at 11 39 12 AM](https://github.com/user-attachments/assets/86d95fae-4a49-4f3f-8293-d1d98147fd76)

**Comment by connorscarolyns** [comment]
I agree. An accessibility panel is the way to go for supporting users with conflicting needs. Can we make sure this doesn't get lost for a future enhancement? We'll need some design research done to see how this can be added to a standard Canada.ca page, which has a strict format. Likely just a link that opens a modal so we don't get into messing with page structure.

**Comment by connorscarolyns** [comment]
Design systems draft accessibility panel:
<img width="720" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd012Q0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--12f41936d22115020b0cef88c2baf5ddd8def56a/image.png" alt="image.png" />
